tools/build_conda.py

```diff
@@ -50,7 +50,7 @@ def move(self, src, dst):
     # Depending on the platform, directories have different names.
     if sys.platform == "win32":
         lib_dest = 'lib'
-        dll_src = 'bin'
+        dll_src = os.path.join("lib", "scipp")
         dll_dest = os.path.join("bin")
```

Comment:
Need to revert this line that I changed this morning?

Reply:
I think this is correct as the DLLs are now being built in a different location, so we need to copy from that new location?

Reply:
Yes `dll_src` is correct, but `dll_dest` needs to revert (you may not have this commit locally) to not use `"bin"`.